.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2965,7 +2972,7 @@ stages:
     timeoutInMinutes: 60 #default value
 
     pool:
-      name: azure-managed-linux-x64-1
```

Comment:
Guessing that this one was missed in https://github.com/DataDog/dd-trace-dotnet/pull/7446?

Reply:
Just to make sure it isn't different for a specific reason the others were on `-2` and being changed to `-1`, this one is on `-1` and staying on `-1`